examples/workflows/script_annotations_artifacts_oss_new.py

```diff
@@ -0,0 +1,38 @@
+try:
+    from typing import Annotated  # type: ignore
+except ImportError:
+    from typing_extensions import Annotated  # type: ignore
+
+from hera.shared import global_config
+from hera.workflows import Workflow, script
+from hera.workflows.artifact import Artifact, OSSArtifact
+from hera.workflows.steps import Steps
+
+global_config.experimental_features["script_annotations"] = True
+global_config.experimental_features["script_runner"] = True
+
+
+@script()
+def read_artifact(
+    my_artifact: Annotated[
+        str,
+        OSSArtifact(
+            name="my_artifact",
+            path="tmp/file",
+            access_key_secret={"name": "my-oss-credentials", "key": "secretKey"},
+            bucket="test-bucket-name",
+            create_bucket_if_not_present=True,
+            endpoint="http://oss-cn-hangzhou-zmf.aliyuncs.com",
+            key="test/mydirectory/",
+            lifecycle_rule={"name": "my-oss-rule", "key": "ruleKey"},
+            secret_key_secret={"name": "my-oss-credentials", "key": "secretKey"},
+            security_token="oss-token",
+        ),
+    ]
+) -> str:
+    return my_artifact
```

Comment:
Will this return work by default if someone does not use the experimental script runner?

Reply:
If someone's not using the runner then I think this would return an empty string? Not using the runner with Artifact annotations seems to be the wrong thing to be doing